flaml/__init__.py

```diff
@@ -2,6 +2,7 @@
 from flaml.automl import AutoML, logger_formatter
 from flaml.onlineml.autovw import AutoVW
 from flaml.version import __version__
+from flaml.utils import *
```

Comment:
```suggestion
try:
    from flaml.utils import *
except ImportError:
    # In case some import from sklearn breaks, don't make `import flaml` fail
    pass
```